main.py

```diff
@@ -179,16 +179,24 @@ def send_email(request: Request) -> Tuple[str, int]:
         )
         return message, status_code
     except RequestException as error:
-        notify_error = error.response.json()["errors"][0]
-        status_code = error.response.status_code
         message = "notify request failed"
-        log_error(
-            message,
-            **log_context,
-            notify_error=notify_error,
-            status_code=status_code,
-        )
-        return message, error.response.status_code
+
+        if error.response:
+            notify_error = error.response.json()["errors"][0]
+            status_code = error.response.status_code
+
+            log_error(
+                message,
+                **log_context,
+                notify_error=notify_error,
+                status_code=status_code,
+            )
+
+            return message, error.response.status_code
+
+        log_error(message, **log_context)
+
+        return message, None
```

Comment:
How come this was needed?.

- Have you tried this in a deployed env?. It is probably expects a status code.
- If it is valid, it needs a new test.

Reply:
Reverted as discussed and will raise another card to address the mypy issues and the broken test on main 👍